app/src/main/res/values/strings.xml

```diff
@@ -281,6 +281,17 @@
     <string name="preferences_freestyle_libre_experimental">EXPERIMENTAL: Enable FreeStyle Libre® support</string>
     <string name="preferences_font_dyslexia_experimental">EXPERIMENTAL: Increase readability for users with dyslexia</string>
     <string name="preferences_font_dyslexia">Increase readability for users with dyslexia</string>
+    <string name="preferences_analytics_opt_in">Enable analytics</string>
+    <string name="allow">Allow</string>
+    <string name="optout">Opt-Out</string>
+    <string name="analytics_usage">Analytics usage</string>
+    <string name="analytics_usage_overview">
```

Comment:
@bkerensa could you please add a paragraph to address collecting data for debug puroposes with Crashlytics?